application/core/LSYii_NoUpdateValidator.php

```diff
@@ -26,10 +26,6 @@ class LSYii_NoUpdateValidator extends CValidator
      */
     public function validateAttribute($object, $attribute)
     {
-        if (Yii::app()->user->isScriptUpdateAllowed()) {
```

Comment:
I am not sure about this. How this plays out?

Reply:
This actually reverses this:
https://github.com/LimeSurvey/LimeSurvey/commit/3eacd351e644ffb46ec06c69d3f5112febffa8a6

Reply:
Yes : because is the way to do … 
LSYii_NoUpdateValidatordisable usage of LSYii_NoUpdateValidator for script …

@c-schmitz : why you do https://github.com/LimeSurvey/LimeSurvey/commit/3eacd351e644ffb46ec06c69d3f5112febffa8a6 this break the action on script

LSYii_NoUpdateValidator : don't update an attribute

Need to be done for script

Reply:
PS : without this fix : any LimeSurvey admin can update script …  on all condition.